src/main/java/org/example/homework3/Task3.java

```diff
@@ -0,0 +1,9 @@
+package org.example.homework3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        long a = 12345678912345L;
+        int b = ((int) a);

```

Comment:
unnecessary parentheses
```suggestion
        int b = (int) a;

```